spec/rails_helper.rb

```diff
@@ -5,6 +5,8 @@
 require 'rspec/rails'
 require 'jsonapi/rspec'
 require 'factory_bot'
+require 'coveralls'
```

Comment:
Shouldn't coveralls be on top?

Reply:
Yes, this needs to go on top of the file.

Reply:
Well this works fine!